### PR TITLE
[StbHardware/Timezones] add func setRTCoffset

### PR DESCRIPTION
**Pull request by Dima73**
- some reciever use /proc/stb/fp/rtc_offset  for correct local time

### CODE REVIEW

File: lib/python/Tools/StbHardware.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from fcntl import ioctl
 from struct import pack, unpack
+from time import time, localtime
```
Comment (athoik) [inline lib/python/Tools/StbHardware.py]: Did you miss gmtime here?

**Comment by littlesat** [comment]
I'll wait for the merge until adhoik's comment is checked and we have green light or a fix...

**Comment by littlesat** [comment]
Can I have a better description for what it exactly is doing....?

**Comment by Dima73** [comment]
Sets the difference for your time zone.

**Comment by athoik** [comment]
What about boxes without rtc_offset? How those boxes work?

**Comment by Dima73** [comment]
I do not know 
mine so the time zone is adjusted automatically.

**Comment by littlesat** [comment]
I think e2 compensates for timezone so no need to additionally do it for rtc...

**Comment by Dima73** [comment]
Wnen use only rtc time answer rtc incorrect

**Comment by littlesat** [comment]
Sorry I still don't underatand explain with examples....

**Comment by Dima73** [comment]
Example...
 timezone + 2
date 01.01.1970 02.00
rtc time 01.01.1970 00.00
When use rtc time ase2 time need first add rtc_offset((localtime(time()).tm_hour-gmtime(time()).tm_hour)*3600)

**Comment by littlesat** [comment]
Need to finetune this later....

**Pull request merged**